[12-Mar-2012 01:11:18] PHP Fatal error:  Uncaught exception 'Zend_Db_Statement_Mysqli_Exception' with message 'Mysqli prepare error: Query execution was interrupted' in /home/zapato/public_html/application/libraries/Zend/Db/Statement/Mysqli.php:77
Stack trace:
#0 /home/zapato/public_html/application/libraries/Zend/Db/Statement.php(115): Zend_Db_Statement_Mysqli->_prepare('SET time_zone =...')
#1 /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Mysqli.php(405): Zend_Db_Statement->__construct(Object(Zend_Db_Adapter_Mysqli), 'SET time_zone =...')
#2 /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Abstract.php(467): Zend_Db_Adapter_Mysqli->prepare('SET time_zone =...')
#3 /home/zapato/public_html/application/modules/Core/Bootstrap.php(112): Zend_Db_Adapter_Abstract->query('SET time_zone =...')
#4 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(256): Core_Bootstrap->_initDb()
#5 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(207): Engine_Applic in /home/zapato/public_html/application/libraries/Zend/Db/Statement/Mysqli.php on line 77
[12-Mar-2012 01:11:19] PHP Fatal error:  Uncaught exception 'Zend_Db_Adapter_Mysqli_Exception' with message 'Can't connect to local MySQL server through socket '/var/lib/mysql/mysql.sock' (2)' in /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Mysqli.php:352
Stack trace:
#0 /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Abstract.php(448): Zend_Db_Adapter_Mysqli->_connect()
#1 /home/zapato/public_html/application/modules/Core/Bootstrap.php(112): Zend_Db_Adapter_Abstract->query('SET time_zone =...')
#2 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(256): Core_Bootstrap->_initDb()
#3 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(207): Engine_Application_Bootstrap_Abstract->_executeResource('db')
#4 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(150): Engine_Application_Bootstrap_Abstract->_bootstrap(NULL)
#5 /home/zapato/public_html/application/libraries/Engine/Application.php(149): Engine_Applica in /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Mysqli.php on line 352
[12-Mar-2012 01:11:46] PHP Fatal error:  Uncaught exception 'Zend_Db_Adapter_Mysqli_Exception' with message 'Can't connect to local MySQL server through socket '/var/lib/mysql/mysql.sock' (2)' in /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Mysqli.php:352
Stack trace:
#0 /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Abstract.php(448): Zend_Db_Adapter_Mysqli->_connect()
#1 /home/zapato/public_html/application/modules/Core/Bootstrap.php(112): Zend_Db_Adapter_Abstract->query('SET time_zone =...')
#2 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(256): Core_Bootstrap->_initDb()
#3 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(207): Engine_Application_Bootstrap_Abstract->_executeResource('db')
#4 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(150): Engine_Application_Bootstrap_Abstract->_bootstrap(NULL)
#5 /home/zapato/public_html/application/libraries/Engine/Application.php(149): Engine_Applica in /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Mysqli.php on line 352
[12-Mar-2012 01:12:10] PHP Fatal error:  Uncaught exception 'Zend_Db_Adapter_Mysqli_Exception' with message 'Can't connect to local MySQL server through socket '/var/lib/mysql/mysql.sock' (2)' in /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Mysqli.php:352
Stack trace:
#0 /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Abstract.php(448): Zend_Db_Adapter_Mysqli->_connect()
#1 /home/zapato/public_html/application/modules/Core/Bootstrap.php(112): Zend_Db_Adapter_Abstract->query('SET time_zone =...')
#2 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(256): Core_Bootstrap->_initDb()
#3 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(207): Engine_Application_Bootstrap_Abstract->_executeResource('db')
#4 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(150): Engine_Application_Bootstrap_Abstract->_bootstrap(NULL)
#5 /home/zapato/public_html/application/libraries/Engine/Application.php(149): Engine_Applica in /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Mysqli.php on line 352
[12-Mar-2012 01:12:41] PHP Fatal error:  Uncaught exception 'Zend_Db_Adapter_Mysqli_Exception' with message 'Can't connect to local MySQL server through socket '/var/lib/mysql/mysql.sock' (2)' in /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Mysqli.php:352
Stack trace:
#0 /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Abstract.php(448): Zend_Db_Adapter_Mysqli->_connect()
#1 /home/zapato/public_html/application/modules/Core/Bootstrap.php(112): Zend_Db_Adapter_Abstract->query('SET time_zone =...')
#2 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(256): Core_Bootstrap->_initDb()
#3 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(207): Engine_Application_Bootstrap_Abstract->_executeResource('db')
#4 /home/zapato/public_html/application/libraries/Engine/Application/Bootstrap/Abstract.php(150): Engine_Application_Bootstrap_Abstract->_bootstrap(NULL)
#5 /home/zapato/public_html/application/libraries/Engine/Application.php(149): Engine_Applica in /home/zapato/public_html/application/libraries/Zend/Db/Adapter/Mysqli.php on line 352
[14-Mar-2012 13:17:27] PHP Parse error:  syntax error, unexpected '<' in /home/zapato/public_html/application/modules/Review/widgets/profile-reviews/index.tpl on line 129
